windows/devices/uwb/UwbConnector.cxx

```diff
@@ -736,11 +740,15 @@ UwbConnector::HandleNotifications(std::stop_token stopToken)
                     }
                     // Check if the call requires a larger buffer.
                 } else if (lastError == ERROR_MORE_DATA || lastError == ERROR_INSUFFICIENT_BUFFER) {
-                    LOG_VERBOSE << logPrefix << "insufficient buffer, " << bytesRequired << " bytes required, current size " << std::size(uwbNotificationDataBuffer) << " bytes";
+                    PLOG_VERBOSE << logPrefix << "insufficient buffer, " << bytesRequired << " bytes required, current size " << std::size(uwbNotificationDataBuffer) << " bytes";
                     const UWB_NOTIFICATION_DATA& notificationDataPartial = *reinterpret_cast<UWB_NOTIFICATION_DATA*>(std::data(uwbNotificationDataBuffer));
                     bytesRequired = std::max(notificationDataPartial.size, static_cast<uint32_t>(minimumNotificationSize));
                     // Attempt to retry the ioctl with the appropriate buffer size, which is now held in bytesRequired.
                     continue;
+                } else if (lastError == ERROR_DRIVER_PROCESS_TERMINATED) {
+                    PLOG_ERROR << logPrefix << "driver process terminated, hr=" << std::showbase << std::hex << hr;
+                    handleDriver.reset();
+                    break; // for({1,2})

```

Comment:
These comments are referring to the old loop exit condition. Since there's only 1 for loop here, it might be fine to just say `// for()`.

```suggestion
                    break; // for()
```

We might just do this in another PR though, so feel free to close this without addressing it, I just wanted to point it out. They're both wrong and inconsistent ({0,1}, {0,2}, I think both done by me), so we should fix it.

Reply:
Sounds good to leave it for another PR - I just copied those comments without thinking about them and I agree with your suggestion